### PR TITLE
#5: update readme, add hex sticker

### PR DESCRIPTION
**Pull request by kaz462**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiraldev/devel/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/devel/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/cran-release/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,257 +11,30 @@
...
-    from the wider community - see [this
-    page.](https://pharmaverse.github.io/admiral/main/CONTRIBUTING.html#type-1-contribution-with-code).
-    Please use the _“good first issue”_ (ideal for new starters) &
-    _“help wanted”_ (ideal for more experienced contributors) issue
-    labels.
+# Installation
 
-## Quick Start Guide for Template
+The package is available from CRAN and can be installed by running
+`install.packages("pharmaversesdtm")`. To install the latest development
```
Comment (manciniedoardo) [inline README.md:33]: Change to `install.packages("pharmaverseadam")`

File: README.Rmd [hunk 2]
```diff
@@ -40,100 +40,30 @@ admiralci_homepage <- "https://pharmaverse.github.io/admiralci"
...
-- `r link("What is included in the template", "#what-is-included-in-the-template")`
-- `r link("Package Extension Guidance", "#package-extension-guidance")`
-- `r link("Quick Start Guide for Template", "#quick-start-guide-for-template")`
+# Purpose
 
-## Extension Expectations 
+To provide a one-stop-shop for ADaM test data in the pharmaverse family of packages. This includes test ADaM data created from {admiral} templates. 
 
-To be included as an `{admiral}` package extension we expect developers to adhere to the following standards:
+There is an integration test that runs {admiral} templates and compares them with the stored version. If differences are found, it then triggers an action: either accepting the difference and storing the new ADaM, or rejecting it and identifying the cause of the differences in the templates.
```
Comment (manciniedoardo) [inline README.Rmd:57]: Maybe this sentence could be a little clearer. I would suggest going from: 

_There is an integration test that runs {admiral} templates and compares them with the stored version. If differences are found, it then triggers an action: either accepting the difference and storing the new ADaM, or rejecting it and identifying the cause of the differences in the templates._

to:

_The ADaM contents of this package is automatically populated by a CICD action that executes `{admiral}` templates and saves the resulting dataset here. The ADaM datasets in `{pharmaversesdtm}` are updated any time the templates are updated in `{admiral}`_

**Review by manciniedoardo — CHANGES_REQUESTED** [review]
Hex looks good, just suggested some changes to wording. Thanks!

File: README.md [hunk 3]
```diff
@@ -13,257 +11,29 @@
...
-family.
+To provide a one-stop-shop for ADaM test data in the pharmaverse family
+of packages. This includes test ADaM data created from {admiral}
+templates.
 
-_Note: The ordering numbers below are suggested but don’t all need to
-strictly be followed in this sequence._
+The ADaM contents of this package is automatically populated by a CICD
+action that executes `{admiral}` templates and saves the resulting
+dataset here. The ADaM datasets in `{pharmaversesdtm}` are updated any
```
Comment (manciniedoardo) [inline README.md:26]: Apologies just noticed this should say `{pharmaverseadam}`!

**Review by manciniedoardo — CHANGES_REQUESTED** [review]
One final typo sorry

File: README.Rmd [hunk 4]
```diff
@@ -40,100 +40,31 @@ admiralci_homepage <- "https://pharmaverse.github.io/admiralci"
...
+Test data (ADaM) for the pharmaverse family of packages
 
-- `r link("Extension Expectations", "#extension-expectations")`
-- `r link("What is included in the template", "#what-is-included-in-the-template")`
-- `r link("Package Extension Guidance", "#package-extension-guidance")`
-- `r link("Quick Start Guide for Template", "#quick-start-guide-for-template")`
+# Purpose
 
-## Extension Expectations 
+To provide a one-stop-shop for ADaM test data in the pharmaverse family of packages. This includes test ADaM data created from {admiral} templates.
```
Comment (bundfussr) [inline README.Rmd:55]: Does "{admiral} template" refer to the templates in admiral core only or does it include the templates in the extension packages {admiralonco}, {admiralophta}, {admiralvaccines}, ...?

If it includes extension packages, we may need to add something regarding the requirements for the CI/CD workflow to process them.

**Review by bundfussr — COMMENTED** [review]
The `Imports:` field in `DESCRIPTION` should be updated.

File: README.Rmd [hunk 5]
```diff
@@ -40,100 +40,31 @@ admiralci_homepage <- "https://pharmaverse.github.io/admiralci"
...
 
-To be included as an `{admiral}` package extension we expect developers to adhere to the following standards:
 
-* `r dyn_link("Code of Conduct", admiral_homepage, "CODE_OF_CONDUCT.html")`
-* `r dyn_link("Manifesto", admiral_homepage, "index.html?q=manifest#admiral-manifesto")`
-* `r dyn_link("Programming Strategy", admiraldev_homepage, "articles/programming_strategy.html")`
-* `r dyn_link("Development Process", admiraldev_homepage, "articles/development_process.html")`
-* Follow consistent workflow checks
-* A CRAN Release means 90% or greater test coverage
+The ADaM contents of this package is automatically populated by a CICD action that executes `{admiral}` templates and saves the resulting dataset here. The ADaM datasets in `{pharmaversesdtm}` are updated any time the templates are updated in `{admiral}`.
```
[the remark this answers is not in the document]
Comment (kaz462) [reply README.Rmd:58]: source datasets updates could be one of the reasons that leads to templates updates?

File: README.Rmd (hunk 4 above)
Comment (kaz462) [reply README.Rmd:55]: I think it includes admiral extension packages as well. @manciniedoardo will we have CI/CD ready for extension packages in this release? if not, @bundfussr can we add more details when the workflow is ready?

Comment (manciniedoardo) [reply README.Rmd:55]: initial setup is only for admiral but then extension packages can be included too in the near future

File: README.Rmd (hunk 5 above)
[the remark this answers is not in the document]
Comment (bundfussr) [reply README.Rmd]: I think the ADaMs should be recreated if
- the template in admiral is updated or
- the source data in pharmaversesdtm is updated

Then pharmaversesdtm, pharmaversadam, and admiral are in sync.
Do yo agree?

File: README.Rmd (hunk 4 above)
Comment (bundfussr) [reply README.Rmd:55]: > if not, @bundfussr can we add more details when the workflow is ready?

Yes, of course.

File: README.Rmd (hunk 5 above)
[the remark this answers is not in the document]
Comment (kaz462) [reply README.Rmd]: updated

File: README.Rmd (hunk 4 above)
Comment (kaz462) [reply README.Rmd:55]: thank you both! a separate issue created: https://github.com/pharmaverse/pharmaverseadam/issues/8

**Review by manciniedoardo — APPROVED** [review]

**Review by bundfussr — APPROVED** [review]
